src/Microsoft.Extensions.Configuration.AzureAppConfiguration/OfflineFileCache.cs

```diff
@@ -285,12 +222,15 @@ private void EnsureOptions(AzureAppConfigurationOptions azconfigOptions)
 
                 _scopeToken = sb.ToString();

```

Comment:
It looks like scopetoken is dependent on method local variables. I don't think it can be a member.